lib/runcible/response.rb

```diff
@@ -23,6 +23,10 @@ def body
       @parsed_body
     end
 
+    def to_hash
+      self.parsed_body.try(:to_hash)
```

Comment:
previous design was bad, and was extending the parsed response class.  In this case it woudl have been 'extending' the nil class